src/components/ChatLog.jsx

```diff
@@ -0,0 +1,46 @@
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+
+const ChatLog = (props) => {
+  const chatentrycomponents = props.entries.map((entries, index) => {
+    return (
+      <ul key={index}>
+        <ChatEntry
+          id={entries.id}
+          sender={entries.sender}
+          body={entries.body}
+          timeStamp={entries.timeStamp}
+          liked={entries.liked}
+          onLiked={props.onLiked}
+          localSender={props.localSender}
+        ></ChatEntry>
```

Comment:
Since the names of the keys on `entry` are the same as the names your are setting on `ChatEntry` attributes/you are using all of the values in `entry` you could do something something like this to save you a few keystrokes: 

```javascript
  const chatComponents = entries.map((entry) => {
    return(
      <ChatEntry
        {...entry}
        onLikeToggle={onLikeBtnToggle}
        key={entry.id}
      />
    );
  });
```
Just recognize that with this method it is not as explicit as what you have. Something to consider!